ChemistryLib/PhreeqcKernel.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <map>
+#include <vector>
+
+#include "ChemicalSolverInterface.h"
+#include "PhreeqcKernelData/AqueousSolution.h"
+
+#include "ThirdParty/iphreeqc/src/src/phreeqcpp/Phreeqc.h"
```

Comment:
If you move the implementations into cpp, this header file could be moved to cpp/replaced with fwd-decls.

Reply:
May be not, even moving the implementation file. This header file, I think, is required for the declaration of the base class `Phreeqc`...